doc/source/conf.py

```diff
@@ -298,8 +303,39 @@
     for page in moved_api_pages
 }
 
+
+common_imports = """\
```

Comment:
Previously we had this at the top of whatsnew.rst. Now it has to be in every `whatsnew/v0.x.x.txt`, as they are no longer literally included below this. Usage is just `{{ common_imports }}`